openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -170,7 +171,11 @@ def _create_jwt(
     # Default scopes should only contain non-privileged data.
     # Do not be misled by the fact that `email` and `profile` are default scopes. They
     # were included for legacy compatibility, even though they contain privileged data.
+    # The scope `user_id` must be added for requests with grant_type password.
     scopes = scopes or ['email', 'profile']
+    if grant_type == Application.GRANT_PASSWORD:
```

Comment:
Do we need something like the following? Do we want a test to ensure it isn't added twice? A set vs list would be nice, but that would probably be too much work.
```suggestion
    if grant_type == Application.GRANT_PASSWORD and 'user_id' not in scopes:
```

Reply:
I just pushed a fix for this before looking at the review, now looking at the ADR suggestions.